templates/zerver/help/signing-out.md

```diff
@@ -0,0 +1,29 @@
+## Logging out of Zulip
+
+   
+When you're done using Zulip, it's easy to log out.
+   
+
+### Logging out on a browser:
+  
+1. Click on the settings gear ( ![Gear](/static/images/help/cog.png) ) in the top right.  
+
+2. Click **Log Out**.
+  
+![Desktop](/static/images/help/desktop_settings_menu_logout_highlighted.png)
+
+  
+
+### Logging out of the iOS app:  
+
+1. Tap the menu icon in the top left ( ![Hamburger](/static/images/help/hamburger_icon.jpg) ) of your screen.
```

Comment:
We should move the image over to immediately "menu icon" so that it's consistent with the Android version below :)